docs/content/ingestion/stream-pull.md

```diff
@@ -252,6 +256,12 @@ then it can store segments with the same datasource, time interval, and version
 
 You can use type `numbered` similarly. Note that type `none` is essentially type `linear` with all shards having a fixed `partitionNum` of 0.
 
+#### IndexSpec
+
+|Field|Type|Description|Required|
+|-----|----|-----------|--------|
+|bitmap|String|Set to `roaring` to use Roaring Bitmaps. Set to `concise` to use Concise bitmaps.|no (default = concise)|
```

Comment:
according to the (code)[https://github.com/b-slim/druid/blob/62ba9ade37f2ab96d3894a0eb622ed5e4f290a03/processing/src/main/java/io/druid/segment/IndexSpec.java#L81-L82] it can be set via runtime.property file, but it seems to be deprecated, can we add this to the doc ?